ansible_rulebook/engine.py

```diff
@@ -443,28 +449,35 @@ def stop(self):
 
     async def execute(self):
         for key in list(self.actions.keys()):
+            if key.startswith("run_playbook"):
+                action = "run_playbook"
```

Comment:
Can you store `action` and `run_method` into the `self.actions` via `action_args`, just like what you did on `async_condition`? so here we don't need these comparisons, just pop up and run them.